examples/websocket-relay/relay-app/src/lib/workflow.ts

```diff
@@ -1,16 +1,18 @@
+import { CID } from "multiformats";
```

Comment:
```suggestion
import { CID } from "multiformats/cid";
```

for now i really need the CID instance to be able to serialize tasks as dag-cbor